src/tools/opt-dist/src/tests.rs

```diff
@@ -102,13 +103,19 @@ llvm-config = "{llvm_config}"
         "tests/incremental",
         "tests/mir-opt",
         "tests/pretty",
+        "tests/run-make/glibc-symbols-x86_64-unknown-linux-gnu",
```

Comment:
Totally agree that having this guard is beneficial!

However, it may cause a problem to opt-dist users outside rust-lang CI who don't care if their rustc artifacts use newer glibc symbols.
Just FYI, it is a bit too minor that I don't think it is worth of a separate issue.

Reply:
I mean, `opt-dist` is primarily used for rust-lang's CI. Any support of it for external users is best-effort. And supporting also the tests is completely out of scope. Tests should now be opt-in after https://github.com/rust-lang/rust/pull/139962, so it should be fine.

Reply:
Yeah I knew that, and that was why I left a comment here only. No need to do anything on your side and sorry for the noise 😞.